simapp/helpers/test_helpers.go

```diff
@@ -19,11 +19,15 @@ const (
 )
 
 // GenTx generates a signed mock transaction.
+// Deprecated - use GenSignedMockTx instead.
```

Comment:
Dont think this needs to be marked as deprecated. Once teams upgrade they will see the api break.

Reply:
Sure, removed